src/.vuepress/layouts/Landing.vue

```diff
@@ -86,44 +86,44 @@
 				<div class="list">
 					<div class="column">
 						<div v-fade-in class="item"><div class="content">
-							<img src="/top-features-note.png" class="img">
+							<img src="/top-features-note.png" class="img" alt="Note">
 							<div class="title">{{ frontmatter.sections.features.list.note.title }}</div>
 							<div class="description">{{ frontmatter.sections.features.list.note.description }}</div>
 						</div></div>
 						<div v-fade-in class="item"><div class="content">
-							<img src="/top-features-reaction.png" class="img">
+							<img src="/top-features-reaction.png" class="img" alt="Reaction">
 							<div class="title">{{ frontmatter.sections.features.list.reaction.title }}</div>
 							<div class="description">{{ frontmatter.sections.features.list.reaction.description }}</div>
 						</div></div>
 						<div v-fade-in class="item"><div class="content">
-							<img src="/top-features-theme.png" class="img">
+							<img src="/top-features-theme.png" class="img" alt="Theme">
 							<div class="title">{{ frontmatter.sections.features.list.theme.title }}</div>
 							<div class="description">{{ frontmatter.sections.features.list.theme.description }}</div>
 						</div></div>
 						<div v-fade-in class="item"><div class="content">
-							<img src="/top-features-charts.png" class="img">
+							<img src="/top-features-charts.png" class="img" alt="Charts">
```

Comment:
Suggestion:
Example image for Charts future

Reply:
I think writing "sample image" is unnecessary information as it should be obvious from HTML semantics that it is an image. So in this case I think just "charts" is fine.